### DISCUSSION TITLE
更新最新版3.1.0后，歌词界面黑乎乎

### DISCUSSION THREAD

**Discussion by 98Protein** [amll-dev/applemusic-like-lyrics#239] [General]
Q：更新至最新版3.1.0后，歌词界面黑乎乎，原来用着好好的，更新之后就出现这个情况了，在NCM插件管理中界面也是白乎乎的。
网易云版本2.10.12
BNCM版本1.3.2
![image](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/54225552/80d8a4f3-10bf-46ae-9c38-f1ebbf283da4)

**Comment by Zellon0w0** [comment]
Q:同样的问题，且拓展设置界面也无法正常打开，其他拓展正常
![image](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/162297823/5d9365c8-9e6e-45ec-8c09-aa242b9b30ed)
![image](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/162297823/bc9fce76-eaef-433b-9219-581e22a9519b)

**Comment by MK0727** [comment]
我也是遇到了这个情况，歌词页面黑乎乎一片，什么也没有，然后拓展设置没办法打开

**Comment by 98Protein** [comment]
我这边已解决，请注意BNCM的安装路径是否在网易云的根目录中，我当时发现网易云在另一个目录，我就重装到与BNCM同路径的目录中之后，新版的applemusic-like-lyrics“类苹果歌词”就没有问题了，歌词页、设置页都能正常显示了的。
